.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -7,15 +7,10 @@ assignees: ''
 
 ---
 
-## Compiler version
-
-``` bash
-# get compiler version via command line
-scalac -version
-```
-
 ## Minimized code
 
+Using Scala (add version here, if you're not sure check with `sbt "print scalaVersion"`),
+
```

Comment:
We encourage programmers to minimize, using SBT for minimization is awkward. I think at least we should list `scalac` as a way to get compiler version.

Reply:
It's much easier to minimize by using your normal build tool than by trying to replace it by scalac, and I think that if they're using scalac by hand, they know which version they've installed, but feel free to add a suggestion with the text you'd like to add here.

Reply:
(https://github.com/scala/bug/blob/master/.github/ISSUE_TEMPLATE/bug.md doesn't list any command and it doesn't seem to be a problem)

Reply:
If you want to add `sbt` and/or `mill` , maybe just add more lines to the code block.

Reply:
Since the current situation is confusing (cf https://github.com/lampepfl/dotty/issues/11211), adding more lines of code is likely to make it more confusing I think, maybe there's a better way to describe things but I would rather remove things that add more things.

Reply:
If a reporter only uses `sbt`, I think it's fair enough to add `sbt "print scalaVersion"` as a way to get compiler version. I think we can trust programmers who bother to report a bug can figure out the version if we give enough tips (I assume most of them are professional programmers). 

Meanwhile, if they don't know `scalac` before, it's good to know and learn that as a Scala programmer --- a reward for bug report :)

Reply:
OK, I've tried to find a wording which keeps both commands without being too confusing.

Reply:
Thanks for taking care 👍